package.json

```diff
@@ -76,7 +76,7 @@
   },
   "dependencies": {
     "@aws-sdk/credential-providers": "^3.535.0",
-    "@govuk-one-login/di-ipv-cri-common-express": "6.3.0",
+    "@govuk-one-login/di-ipv-cri-common-express": "^6.2.2",
```

Comment:
Why have we downgraded? I'm about to have to upgrade to 6.4.0 so just want to check there's no issues

Reply:
It was causing this failure in pre merge checks, so I reverted it back to the version that was in place when I made the branch:

Error: Cannot find module 'lodash.frompairs'
Require stack:
- /home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/user-ip-address.js
- /home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/axios.js
- /home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/index.js
- /home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/index.js
- /home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/src/app.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
    at Module._load (node:internal/modules/cjs/loader:981:[27](https://github.com/govuk-one-login/ipv-cri-f2f-front/actions/runs/9115497622/job/25061961016?pr=398#step:8:28))
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/user-ip-address.js:2:19)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Module.require (node:internal/modules/cjs/loader:12[31](https://github.com/govuk-one-login/ipv-cri-f2f-front/actions/runs/9115497622/job/25061961016?pr=398#step:8:32):19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/user-ip-address.js',
    '/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/axios.js',
    '/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/lib/index.js',
    '/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/node_modules/@govuk-one-login/di-ipv-cri-common-express/src/index.js',
    '/home/runner/work/ipv-cri-f2f-front/ipv-cri-f2f-front/src/app.js'
  ]
}

Reply:
Hmm have we tried deleting package-lock and node_modules and re-installing?

Reply:
If this version has been merged it must work because it would have had to pass the pre merge checks at some point...?

Reply:
Yes deleting and reinstalling worked thanks :)